docs/agents/sematext-agent/windows-installation.md

```diff
@@ -0,0 +1,155 @@
+title: Installing Sematext Agent on Windows
+
+Sematext Agent collects a plethora of metrics about Windows hosts (cpu, memory, disk, network, processes). The installer is responsible to install the Sematext Agent, configuring it, and starting it as a Windows service.
+
+Installation can be done either by using the GUI installer or through the command line.
+
+## Pre-requirements
+
+Before starting, please check the [platform support policy](https://sematext.com/docs/agents/sematext-agent/platform-support-policy/) page to make sure your Windows platform is supported.
+
+To see your Windows host metrics in Sematext Cloud, you need to [create an Infra App](https://sematext.com/docs/monitoring/infrastructure/#create-an-infra-app) and select the Windows environment. This will provide you with an Infra App token, which is essential for establishing a secure link between Sematext Cloud and Sematext Agent.
+
+## Install from GUI
+
+1. Download the [Sematext Agent installer](https://todo-add-public-installer-repo)
+2. Run the installer as an Administrator
```

Comment:
I guess no need to this step. I have tested with Windows 10, It worked fine. I asked for admin permession when it's needed. So we can remove this step, I think.

See the Install button;
![Screenshot from 2024-04-24 18-34-17](https://github.com/sematext/docs/assets/20258973/b0669cf7-26f8-4091-b97d-725cf80e625a)